source/index.html.md.erb

```diff
@@ -1,6 +1,6 @@
 ---
-title: Analytical Platform User Guidance
-weight: 10
+title: Introduction
+weight: 20
 ---
 
-<%= partial 'documentation/index' %>
+<%= partial 'documentation/introduction' %>
```

Comment:
Could we change the partial from `introduction.md` to `index.md` for consistency?

Reply:
👍 done